src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ CHANGELOG
 8.0
 ---
 
+ * Remove `CsvEncoder::ESCAPE_CHAR_KEY` constant and escape character functionality
+ * Remove `CsvEncoderContextBuilder::withEscapeChar()` method
+ * Remove `symfony/deprecation-contracts` dependency
```

Comment:
this does not need to be mentioned in the changelog. The dependency on `symfony/deprecation-contracts` is an implementation detail.